.eslintrc.json

```diff
@@ -1,21 +1,31 @@
 {
   "root": true,
-  "extends": ["airbnb", "plugin:jest/recommended"],
   "env": {
     "browser": true,
     "jest": true
   },
+  "extends": [
+    "airbnb",
+    "plugin:jest/recommended",
+    "plugin:prettier/recommended"
+  ],
   "globals": {
     "COMMITHASH": "readonly"
   },
-  "plugins": ["jest"],
+  "plugins": [
+    "jest",
+    "prettier",
+    "simple-import-sort"
+  ],
   "rules": {
     "jsx-a11y/anchor-is-valid": [
       "error",
       {
         "specialLink": ["to"]
       }
     ],
-    "react/jsx-props-no-spreading": ["off"]
+    "react/jsx-props-no-spreading": "off",
+    "prettier/prettier": "error",
```

Comment:
The `plugin:prettier/recommended` should already include this config-change.

Reply:
Ah, yeah. Right. Thank you Lars 🙂